environment/env.go

```diff
@@ -133,11 +143,41 @@ func SetupEnv() {
 	}
 }
 
+func lookupLocalhostIP(domain string) string {
```

Comment:
this is really only parameterized for testing. is there a better way to do that?